extensions/sql-migration/src/wizard/sourceConfigurationPage.ts

```diff
@@ -0,0 +1,323 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import { MigrationWizardPage } from '../models/migrationWizardPage';
+import { MigrationStateModel, StateChangeEvent } from '../models/stateMachine';
+import * as constants from '../constants/strings';
+import { IconPath, IconPathHelper } from '../constants/iconPathHelper';
+
+const headerLeft: azdata.CssStyles = {
+	'border': 'none',
+	'text-align': 'left',
+	'white-space': 'nowrap',
+	'text-overflow': 'ellipsis',
+	'overflow': 'hidden',
+	'border-bottom': '1px solid'
+};
+
+const styleLeft: azdata.CssStyles = {
+	'border': 'none',
+	'text-align': 'left',
+	'white-space': 'nowrap',
+	'text-overflow': 'ellipsis',
+	'overflow': 'hidden',
+};
+export class SourceConfigurationPage extends MigrationWizardPage {
+	private _view!: azdata.ModelView;
+	private _databaseSelectorTable!: azdata.DeclarativeTableComponent;
+	private _dbNames!: string[];
+	private _dbCount!: azdata.TextComponent;
+
+	constructor(wizard: azdata.window.Wizard, migrationStateModel: MigrationStateModel) {
+		super(wizard, azdata.window.createWizardPage(constants.SOURCE_CONFIGURATION, 'MigrationModePage'), migrationStateModel);
```

Comment:
Change the title of page to databases for migration

Reply:
So removing the (s) from database(s)?

Reply:
Generally it's preferred to just use plurals whenever there can be 1+ of something - (s) is difficult to translate into other languages being one reason.